app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    long_time_list = {}
+
+    def inner(*args, **kwargs):
+
+        list_args = [*args]
+        if str(list_args) in long_time_list:
```

Comment:
you perform redundant operations: `args` is a tuple, so you can just store it as key in dictionary instead of transforming it into list and then string.